htdocs/install/mysql/tables/llx_product_lot.sql

```diff
@@ -21,11 +21,11 @@ CREATE TABLE llx_product_lot (
   rowid integer AUTO_INCREMENT PRIMARY KEY,
   tms timestamp,
   fk_product integer NOT NULL,
-  batch varchar(30) NOT NULL,
-  eatby datetime DEFAULT NULL,
-  sellby datetime DEFAULT NULL,
+  fk_furnisher integer NOT NULL default 0,
+  reflot varchar(30) NOT NULL,
```

Comment:
I don't want to rename any fields related to stock now. Reason is that it will break a lot of thing and we should start the freeze very soon (freeze should occurs in april)
For 4.0, I would like to reduce change to updating table llx_product_lot each time a new lot is known (each time a reception is done).